### ISSUE TITLE
Using Releases in GitHub and Version in package.json

### ISSUE THREAD

**Issue by haeferer** [Brewskey/spark-server#178]
Hi,

you did an amazing work. 
I currently do some testing (using docker based on different version of node, npm etc.) 

To Help supporting these, is it possible to start 

* using versions in package.json 
* and  corresponding releases in GitHub ?

the only way to mark an image as "a specific version to test" is the commit ID ...

If we are start using a real version schema, i can also start pushing images to docker-hub.

So creating a SparkSever Running a specific version will be only a "one-line" like 

```
docker run spark-server:1.0.1
```

**Comment by jlkalberer** [comment]
Yeah, sounds good.  The code is churning so fast right now that it will be a pain to version everything.

I think once we close a few of these current issues we'll put out a release.
